cypress/integration/page_tabs.ts

```diff
@@ -72,11 +72,11 @@ describe("Tabs", () => {
       locationPathEquals(patches.tasks.route);
     });
 
-    it("clicking away from each tab doesn't crash app", () => {
+    it("should be able to toggle between tabs", () => {
       cy.visit(patchRoute);
-      cy.dataCy(patches.changes.btn).click();
+      cy.get(`button[data-cy='${patches.changes.btn}']`).click();
```

Comment:
nit: it could be helpful to wrap this string formatting into a function called `cy.getTab`